### PR TITLE
Update to match current best config and simplify the parameterization…

### PR DESCRIPTION
**Pull request by rfablet**
PR to push "my" current best configuration for SSH mapping. Updates include:
- some updates of the parameters
- specific dropout parameter for phi_r set to 0.
- removal of beta** wieghts in the computation of the loss, replaced by different weights for mse_ssh and mse_gssh (no need to compute on the gradient norm over the training dataset before training)

### CODE REVIEW

**Comment by rfablet** [comment]
Additional commit to update the definition of exp3 config as discussed, ie 10-day lag between training/val/test and 20-day output sequence (when considering the center of each 5-day window) for the -validation and test dataset.

**Review by quentinf00 — APPROVED** [review]
